apps/ngx-env-demo/angular.json

```diff
@@ -1,26 +1,27 @@
 {
-  "$schema": "../../node_modules/@angular/cli/lib/config/schema.json",
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
   "version": 1,
   "newProjectRoot": "projects",
   "projects": {
     "ngx-env-demo": {
       "projectType": "application",
-      "schematics": {
-        "@schematics/angular:application": {
-          "strict": true
-        }
-      },
+      "schematics": {},
       "root": "",
       "sourceRoot": "src",
       "prefix": "app",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@ngx-env/builder:application",
```

Comment:
the idea si to be able to test out schematics generation in the CI so we keep the initial angular.json file and we apply the schematics using `npx ng add @ngx-env/builder` which will update angular.json and add .env.d.ts